core/src/main/resources/db/changelog/logs/ch-add-tipsandtricks-comment-Beshlei.xml

```diff
@@ -33,6 +33,11 @@
     </changeSet>
 
     <changeSet id="Beshlei-2" author="Taras Beshlei">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <foreignKeyConstraintExists foreignKeyName="fk_comment_parent" />
```

Comment:
Why this change and many others like this? Is there another script that is also adding this?

Reply:
H2 database doesn't allow same foreign key name